optimize/self-managed/optimize-deployment/migration-update/camunda-7/3.13-to-3.14.md

```diff
@@ -17,6 +17,11 @@ Here you will find information about:
 - Changes in behavior (for example, due to a new feature)
 - Changes in translation resources
 
+## Versioning
```

Comment:
> Optimize instances of versions 8.5 and below will not be upgradeable to Optimize 3.14.

This is also kind of confusing because there wasn't really such thing as Optimize version 8.5. Wondering if we can present this information in a less ambiguous way 🤔

Reply:
Did we not just do the 8.5 release? We have the upgrade notes [here](https://docs.camunda.io/optimize/self-managed/optimize-deployment/migration-update/3.12_8.4-to-3.13_8.5/) ("Update notes (8.4/3.12 to 8.5/3.13)")

Reply:
What I mean is that the product wasn't forked. 8.5 and 3.13 was the same thing. So technically 8.5 -> 3.14 is possible, it's the data that we use to infer the intended running mode that makes it not possible

Reply:
Will rephrase